Update Authority and Claim definitions (#572)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

See also "Appraisal Policy for Evidence" in {{-rats-arch}}.

Authority:
: The entity that asserts a Claim.
Typically, a Claim is asserted using a cryptographic key to digitally sign the Claim.
A cryptographic key can be a proxy for a human or organizational entity.

Claim:
: A piece of information, in the form of a key-value pair.
See also {{Section 4.2 of -rats-arch}} and {{Section 2 of -jwt}}.

> Within this document a Claim is either Evidence, a Reference Value, or an Endorsement.

Reviewer: ```suggestion
Within this document, Claims appear in Evidence, a Reference Value or an Endorsement.
```
